src/main/java/com/band/api/repository/Band.java

```diff
@@ -0,0 +1,26 @@
+package com.band.api.repository;
+
+import javax.persistence.*;
```

Comment:
In general don't do star imports, remove this line and add each item in one at a time (for all repository files).